index.html

```diff
@@ -0,0 +1,134 @@
+                <a class="drop-link" href="#program">Program</a>
+                <a class="drop-link" href="#join">Join</a>
+                <a class="drop-link" href="#sponsor">Sponsor</a>
+                <a class="drop-link" href="#news">News</a>
+                <a class="drop-link" href="#campaign">CC Campaign</a>
+            </div>
+
+            <div class="conference-title">
+                <h2>"Hello! Sharing world"</h2>
+                <img src="./images/creative-commons.png">
+            </div>
+            <div class="title-intro">
+                <p>A joyful celebration believing in the value of openness and sharing, creating a positive change from people over 80 countries is taking in October, in Korea.</p>
+                <img src="./images/colored-circle.png">
+            </div>
+            <div class="dt-container">
+                <div class="dt-item2"><h4>2015-10-15 (THU) ~ 16 (FRI)</h4></div>
+                <div class="dt-item3"><p>National Stadium of Korea, Art Center Nabi</p></div>  
+              </div>
+        </header>
+
+        <main>
+            <div class="main-program">
+                <h2>Main Program</h2>
+                <hr>
+                <div class="program-container">
+                    <div class="program-item">
+                        <img class="program-img" src="./images/lecture.png">
+                        <p class="program-name">Lecture</p>
+                        <p class="program-desc">Listen to the speakers from various countries about the messages of sharing.</p>
+                    </div>
+                    <div class="program-item">
+                        <img class="program-img" src="./images/exhibition.png">
+                        <p class="program-name">CC Exhibition</p>
+                        <p class="program-desc">Appreciate various creations applying CC license of artists, organized from Center.</p>
+                    </div>
+                    <div class="program-item">
+                        <img class="program-img" src="./images/forum.png">
+                        <p class="program-name">Forum</p>
+                        <p class="program-desc">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                    </div>
+                    <div class="program-item">
+                        <img class="program-img" src="./images/workshop.png">
+                        <p class="program-name">Workshop</p>
+                        <p class="program-desc">Try creating your own work using open source license rather than just watching.</p>
+                    </div>
+                    <div class="program-item">
+                        <img class="program-img" src="./images/ignite.png">
+                        <p class="program-name">CC Ignite</p>
+                        <p class="program-desc">Get opportunities to network with CC affiliates around the world, also after the summit.</p>
```

Comment:
- [ ] In my opinion, it would be better to make sure the program items have the following to match Cindy's design:
  - The same height
  - A reduced background opacity so that we can see behind
  - A hover effect in form of white borders.

Your design:
![Screenshot from 2023-03-16 07-23-39](https://user-images.githubusercontent.com/97834160/225523417-f460d08d-38d9-477b-8984-25adaf1fc9ac.png)
Cindy's:
![Screenshot from 2023-03-16 07-29-04](https://user-images.githubusercontent.com/97834160/225523546-9d9be6ee-7963-4744-b741-7b80daba8a8b.png)

Reply:
All the recommended changes have been made successfully.